components/Navbar/NavItem.js

```diff
@@ -1,10 +1,11 @@
 import NextLink from "next/link";
 import { Button } from "@chakra-ui/react";
+import { FormattedMessage } from "react-intl";
 const Pages = [
-  { title: "Home", route: "/" },
-  { title: "Contact", route: "/Contact" },
-  { title: "Donores", route: "/Donores" },
-  { title: "About", route: "/About" },
+  { title: <FormattedMessage defaultMessage="Home" />, route: "/" },
+  { title: <FormattedMessage defaultMessage="Contact" />, route: "/contact" },
+  { title: <FormattedMessage defaultMessage="Donores" />, route: "/donores" },
```

Comment:
there is a spell error in donores

Reply:
Thanks @0marwa0 , I fixed it along with some misspellings that Mariam told us about